src/kemal/route.cr

```diff
@@ -20,4 +22,25 @@ class Kemal::Route
     end
     true
   end
+
+  # checks if request params contain _method param to override request incoming method
+  def check_for_method_override!(request)
+    request.override_method = request.method
+    if request.method == "POST"
+      params = Kemal::ParamParser.new(self, request).parse_request
+      if params.has_key?("_method") && self.is_override_method_valid?(params["_method"])
+        request.override_method = params["_method"]
+      end
+    end
+  end
+
+  # checks if method contained in _method param is valid one
+  def is_override_method_valid?(override_method)
+    return false unless override_method.is_a?(String)
+    override_method = override_method.upcase
+    if (override_method == "PUT" || override_method == "PATCH" || override_method == "DELETE")
```

Comment:
Agreed on this as well. I' am in a bad habit to write a bit extra code just for clarity purposes so if after few days I came back at this code, I can easily understand what's going on without having to analyse it. Do you want me to update this?